### ISSUE TITLE
[BUG] - Navbar

### ISSUE THREAD

**Issue by omisticstech** [heroui-inc/heroui#2170]
### NextUI Version

2.2.9

### Describe the bug

missing imports for NavbarMenu, NavbarMenuItem,NavbarMenuToggle components

### Steps to Reproduce the Bug or Issue

just open the file in vscode with those codes it will shows error.

### Expected behavior

as a user i expected to work just by copy paste but i got mission import error

### Operating System Version

windows 11

### Browser

Firefox

**Comment by AbdulrahmanHasanato** [comment]
Yes, it is a bug, and as a user, you should indeed copy-paste to work. But it's too simple to fix. If you're using VSCode you can just delete the last letter and re-write it and it will suggest the import🫠

**Comment by wingkwong** [comment]
No longer reproducible.
